src/app/[locale]/(simulation)/(large-layout)/infos/[question]/layout.tsx

```diff
@@ -0,0 +1,29 @@
+import { noIndexObject } from '@/constants/metadata'
+import { getServerTranslation } from '@/helpers/getServerTranslation'
+import { getMetadataObject } from '@/helpers/metadata/getMetadataObject'
+import type { DefaultPageProps } from '@/types'
+import type { PropsWithChildren } from 'react'
+
+export async function generateMetadata({ params }: DefaultPageProps) {
+  const { locale } = await params
+  const { t } = await getServerTranslation({ locale })
+
+  return getMetadataObject({
+    locale,
+    title: t(
+      'Question personnalisée, rejoindre une campagne - Nos Gestes Climat'
+    ),
+    description: t(
+      'Comprenez comment calculer votre empreinte sur le climat en 10min chrono.'
+    ),
+    robots: noIndexObject,
+  })
+}
+
+export default async function Layout({ children }: PropsWithChildren) {
```

Comment:
Not async (same below) + maybe DRY a bit ? You can curry the title and calls a generic function on each layout but that's optional (remove async is not maybe add an eslint rule to detect it)

Reply:
Ah oui je retire l'async et le div qui est inutile ici ; en revanche obligé de créer ces layout.tsx pour pouvoir définir `generateMetadata` pour chaque page